sklearn/cluster/tests/test_optics.py

```diff
@@ -245,6 +246,30 @@ def test_dbscan_optics_parity(eps, min_samples):
     assert percent_mismatch <= 0.05
 
 
+def test_min_samples_min_clusters_float():
```

Comment:
you can modify test_extract_xi to add some tests using float min_samples and min_cluster_size.

Reply:
Thanks